### PR TITLE
Add Select component

### LINKED ISSUE #17: Select box
- [x] Select box can be optionally preceded with a label on top.
- [x] Optional placeholder prop, which sets the select to blank.
- [x] Optional function passed as props to be called `onChange`.
- [x] Each option in the select can have a `disabled` attribute -> Doesn't render it or greys it out.
- [x] Error reporting http://zpl.io/2yBmG3o
  - [x] Red border around box
  - [x] Error message red font below box. Smaller font. 
- [x] Select label has DarkGray color if no value is selected, otherwise LightGray
- [x] Select fontColor is MiddleGray color if no value is selected, otherwise DarkGray.
- [x] Replace native dropdown icon with `@kiwicom/icons/ChevronUp` icon.
- [x] Add tests.
  - [x] Test `onChange` callback.
  - [x] Test rendering via jest snapshots.  

Example:
image

Error reporting:
image

### PR DESCRIPTION
**Pull request by git-toni**
Closes #17

### CODE REVIEW

File: blkn/src/Select/index.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  }
+  select {
+    font-size: 16px;
+    border-radius: 3px;
+    background-color: ${WHITE_COLOR};
+    border: solid 1px ${LIGHTGRAY_COLOR};
+    line-height: 1.25;
+    padding: 12px 16px;
+    color: ${DARKGRAY_COLOR};
+    -moz-appearance: none;
```
Comment (jaroslav-kubicek) [inline blkn/src/Select/index.js]: these prefix variants are not done automatically by `styled-jsx`?

File: blkn/src/Typography/index.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ type Props = {
   size: "large" | "small" | "normal",
   type: "primary" | "secondary" | "attention" | "error",
   children: React.Node,
+  color?: string,
```
Comment (jaroslav-kubicek) [inline blkn/src/Typography/index.js]: is it desired to introduce colour? I thought all possible states should be achievable by combination of `type` and `variant` https://zpl.io/VxBkGZW

Comment (jaroslav-kubicek) [reply blkn/src/Typography/index.js]: I just created PR to fix colors 🙂

Comment (git-toni) [reply blkn/src/Typography/index.js]: How do you make use of `variant` with the `Typography` component?
I think it's good to have a handy set of colors `primary`, `secondary`, etc and maximize its use. But imho having an optional `color` prop makes the `Typography` component much more re-usable at a very low cost. (*End of salespitch*) :smile:

File: blkn/src/Select/index.js (hunk 1 above)
Comment (git-toni) [reply blkn/src/Select/index.js]: I couldn't find it in the documentation, but just confirmed it does by trying it. :+1:

File: blkn/src/Typography/index.js (hunk 2 above)
Comment (Fallion) [reply blkn/src/Typography/index.js]: We'll be bundling Theme support in the future. #35

**Comment by jaroslav-kubicek** [comment]
there is conflict, cc @git-toni

**Pull request merged**